libafl_frida/build.rs

```diff
@@ -9,4 +9,17 @@ fn main() {
     // Force linking against libc++
     #[cfg(unix)]
     println!("cargo:rustc-link-lib=dylib=c++");
+
+    // Build the test harness
+    // clang++ -shared -fPIC -o harness.so harness.cpp
+    #[cfg(unix)]
```

Comment:
should we make it #[cfg(test)]?

Reply:
I tried, but it seems that test is not passed to build.rs. When I run cargo test - if #[cfg(test)] is there - the compilation is skipped. I'm newbie to Rust so if you have a suggestion how to make it work - please let me know

Reply:
When you run tests (at least of a library crate), the current directory is the one of the crate. Placing the so in some other location will complicate running the test. Are you sure?